lang/pt.js

```diff
@@ -62,7 +62,7 @@ export default {
 	"components.input-time-range.endTime": "Hora final",
 	"components.input-time-range.errorBadInput": "{startLabel} precisa ser anterior a {endLabel}",
 	"components.input-time-range.startTime": "Hora de início",
-	"components.interactive.instructions": "Press Enter to interact, Escape to exit",
+	"components.interactive.instructions": "Pressione Enter para interagir, Escape para sair",
```

Comment:
Uses `Enter` and `Escape`. This seems to be common enough I'm just going to leave them